pages.zh/common/behat.md

```diff
@@ -1,9 +1,9 @@
 # behat
 
-> 基于Behaviour-Driven Development的自动化测试PHP框架
-> 更多信息: <https://behat.org>.
+> 基于 Behaviour-Driven Development 的自动化测试 PHP 框架
```

Comment:
Missing a period here?

Reply:
Ah yes, need to fix that on all the files.

Reply:
```suggestion
> 基于 Behaviour-Driven Development 的自动化测试 PHP 框架。
```